ucm2/sof-soundwire/sof-soundwire.conf

```diff
@@ -34,6 +35,11 @@ DefineRegex {
 		Regex " mic:([a-z0-9]+)"
 		String "${CardComponents}"
 	}
+	# FIXME: the case cfg-dmics=0 should fail.
+	Dmics {
+		Regex " cfg-dmics:([0-9]+)"
```

Comment:
Better regex: `Regex " cfg-dmics:([1-9][0-9]*)"` . I think that you can remove the FIXME with it. BTW: Nice regex online tool: https://regex101.com/

Reply:
ack, done